swap_meet/vendor.py

```diff
@@ -1,2 +1,99 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
\ No newline at end of file
+    def __init__ (self, inventory = None):
+        if not inventory:
+            inventory = []
+        self.inventory = inventory
+    
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+    
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        
+        self.inventory.remove(item)
+        return item
+    
+    def get_by_id(self, id):
+        for item in self.inventory:
+            if item.id == id:
+                return item
+        return None    
+    
+    def swap_items(self, other_vendor, my_item, their_item):
+        if my_item in self.inventory and their_item in other_vendor.inventory:
+            self.remove(my_item)
+            other_vendor.add(my_item)
+            
+            other_vendor.remove(their_item)
+            self.add(their_item)
```

Comment:
The `remove()` method y'all wrote returns the item that gets removed. We can use that to our advantage by returning the item directly to our `add()` method. This could look something like the following: 

```suggestion
            self.add(other_vendor.remove(their_item)
```

We could do the same for the previous set of remove and add statements!